test_sets/External_Consent_Status_OFF.txt

```diff
@@ -8,7 +8,7 @@
 ./test_scripts/Polices/External_UCS/External_Consent_OFF/008_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_omitted_user_consent_allowed.lua Check_that_SDL_returns_SUCCESS_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_OFF_and_functional_group_allowed_data_consent_allowed
 ./test_scripts/Polices/External_UCS/External_Consent_OFF/009_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_omitted_user_consent_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_OFF_and_functional_group_disallowed_by_user
 ./test_scripts/Polices/External_UCS/External_Consent_OFF/010_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_omitted_user_consent_omitted.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_OFF_and_functional_group_omitted
-./test_scripts/Polices/External_UCS/External_Consent_OFF/011_ATF_Policies_External_Consent_OFF_through_ignition_cycles_allowed_RPCs.lua Check_that_SDL_uses_value_externalConsentStatus_OFF_when_disallowed_by_external_consent_entities_on_exists_allowed_RPCs_during_ign_cycles
+./test_scripts/Polices/External_UCS/External_Consent_OFF/011_ATF_Policies_External_Consent_OFF_through_ignition_cycles.lua Check_that_SDL_uses_value_externalConsentStatus_OFF_when_disallowed_by_external_consent_entities_on_exists_allowed_RPCs_during_ign_cycles
```

Comment:
@istoimenova, According to previous version of test set it was assumed to have 3 scripts for ignition_cycles:
1) _on -> OFF (allowed RPCs): 011
2) _off -> OFF (disallowed RPCs): 012
3) _on/_off omitted -> OFF (disallowed RPCs): 013

By using new name for 011 script is these 3 cases will be implemented?